source/services/index.js

```diff
@@ -12,7 +12,8 @@ var services = {
     pinterest:     require('./pinterest'),
     facebook:      require('./facebook'),
     twitter:       require('./twitter'),
-    gplus:         require('./gplus')
+    gplus:         require('./gplus'),
+    telegram:      require('./telegram')
```

Comment:
@NickMitin We don’t follow this rule at the moment, so this shouldn’t be done yet